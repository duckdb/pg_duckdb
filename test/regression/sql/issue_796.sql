SET duckdb.force_execution = TRUE;

CREATE TEMP TABLE t2(a INT);
INSERT INTO t2 SELECT i from generate_series(1, 2048) as i; -- works with 2047
SELECT * FROM t2;
