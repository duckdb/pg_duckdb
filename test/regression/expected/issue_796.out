SET duckdb.force_execution = TRUE;
create temp table t2(a int);
insert into t2 select i from generate_series(1, 200000) as i;
select count(*), sum(a), count(*) = 100000 from t2 where a%2 = 0;
 count  |     sum     | ?column? 
--------+-------------+----------
 100000 | 10000100000 | t
(1 row)

